lms/djangoapps/instructor_task/tests/test_tasks.py

```diff
@@ -292,7 +292,7 @@ def test_overriding_non_scorable(self):
         mock_instance = MagicMock()
         del mock_instance.set_score
         with patch(
-                'lms.djangoapps.instructor_task.tasks_helper.module_state.get_module_for_descriptor_internal'
+                'lms.djangoapps.instructor_task.tasks_helper.module_state.get_block_for_descriptor_internal'
         ) as mock_get_module:
```

Comment:
```suggestion
        ) as mock_get_block:
```

There are eight occurrences of this name in this file.